lib/kb_orthofinder/kb_orthofinderImpl.py

```diff
@@ -207,6 +207,15 @@ def propagate_annotation(self, cluster, threshold, plantseed_curation):
     def __init__(self, config):
         #BEGIN_CONSTRUCTOR
         self.workspaceURL = config['workspace-url']
+
+        self.testing = False
```

Comment:
This seems like it might be easy to check in the wrong values in the deploy.cfg and make the app run against the precomputed dataset instead of the users dataset, and probably a regular code release review won't catch it, as I doubt they look at the deploy.cfg. Maybe this should be passed into the individual functions that need it?

Reply:
Here's how another app deals with this https://github.com/kbaseapps/kb_gtdbtk#test-modes

Reply:
But the text at the link says: "However, it does not test the code in the Impl file." This doesn't make sense to me, what is it actually testing?

Reply:
If you read the makefile you can see the command it runs

* https://github.com/kbaseapps/kb_gtdbtk/tree/master/test

It runs stuff in the core directory, but not the server test. This is useful for development as its a shorter test and doesn't care about actually sending real data to the kbase system (and probably doesn't even require a token)